_data/changelogs/component-process-list-accessibility.yml

```diff
@@ -0,0 +1,8 @@
+title: Process list accessibility tests
+type: component
+items:
+ - date: NNNN-NN-NN
+   summary: Added accessibility tests page.
+   affectsGuidance: true
+   githubPr: 3007
+   githubRepo: uswds-site
```

Comment:
**polish:** Format code

```suggestion
   githubRepo: uswds-site
   
```

Reply:
Resolved in [73ddb50](https://github.com/uswds/uswds-site/pull/3007/commits/73ddb50d7c1d23da0cfc280840136c43b0245c75)